openjdk/src/test/java/org/conscrypt/PlatformTestObjectUtil.java

```diff
@@ -0,0 +1,41 @@
+package org.conscrypt;
+
+import java.security.AlgorithmConstraints;
+import java.security.AlgorithmParameters;
+import java.security.CryptoPrimitive;
+import java.security.Key;
+import java.util.Set;
+import javax.net.ssl.SNIMatcher;
+import javax.net.ssl.SNIServerName;
+
+final class PlatformTestObjectUtil {
+    static SNIMatcher newSniMatcher() {
```

Comment:
This should be more specific that this is a failing SNIMatcher, something like `newFailingSniMatcher`.

Reply:
Moved to separate class under testing module.